etpServerExample/MyOwnStoreProtocolHandlers.cpp

```diff
@@ -89,9 +89,16 @@ void MyOwnStoreProtocolHandlers::on_PutDataObjects(const Energistics::Etp::v12::
 		}
 	}
 
+	Energistics::Etp::v12::Protocol::Store::PutDataObjectsResponse objResponse;
```

Comment:
I think you must fill in the map in the response with the keys which have been sent.
The associated values must be empty but the keys must be present.

Reply:
Done.